pysat/tests/classes/cls_instrument_library.py

```diff
@@ -339,20 +384,10 @@ def test_load(self, clean_level, inst_dict):
             test_inst.clean_level = clean_level
             target = 'Fake Data to be cleared'
             test_inst.data = [target]
-            try:
-                test_inst.load(date=date, use_header=True)
-            except ValueError as verr:
-                # Check if instrument is failing due to strict time flag
-                if str(verr).find('Loaded data') > 0:
-                    test_inst.strict_time_flag = False
-                    with warnings.catch_warnings(record=True) as war:
-                        test_inst.load(date=date, use_header=True)
-                    assert len(war) >= 1
-                    categories = [war[j].category for j in range(0, len(war))]
-                    assert UserWarning in categories
-                else:
-                    # If error message does not match, raise error anyway
-                    raise(verr)
+
+            # Make sure the strict time flag doesn't interfere with
+            # the load tests
+            set_strict_time_flag(test_inst, date, raise_error=True)
 
             # Make sure fake data is cleared
             assert target not in test_inst.data
```

Comment:
this is the bit that has changed. `set_strict_time_flag` loads the data (which is why 393 passes). But in some cases the new function will force clean to be 'none', which is a change in behaviour.